client/state/reader/posts/selectors.js

```diff
@@ -47,5 +47,4 @@ export const getPostsByKeys = treeSelect(
 	{ getCacheKey: postKeys => postKeys.map( keyToString ).join() }
 );
 
-export const hasPostBeenSeen = ( state, globalId ) =>
-	!! get( state, [ 'reader', 'posts', 'seen', globalId ] );
+export const hasPostBeenSeen = ( state, globalId ) => !! state?.reader?.posts?.seen?.[ globalId ];
```

Comment:
Assuming that `reader` reducer has been loaded, `posts` and its `seen` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?